rholang/examples/linking/packages/RevSystem.rho

```diff
@@ -0,0 +1,64 @@
+export RevSystem in {
+  import ConditionedToken, SecretName, SignatureCondition in {
+    new result, revTokenCh in {
+      SecretName(result) | for(r <- result) {
+        match r with [revSupplyKey, revSupplyLock] => {
+          ConditionedToken("supply", revSupplyLock, 1000000, revTokenCh) |
```

Comment:
This initial account is 100% made up by me. There definitely need to be discussions around what the initial account and supply are, but these are more governance related then technical.